azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageVersionServlet.java

```diff
@@ -108,6 +113,26 @@ protected void handleGet(HttpServletRequest req, HttpServletResponse resp, Sessi
   @Override
   protected void handlePost(HttpServletRequest req, HttpServletResponse resp, Session session)
       throws ServletException, IOException {
+    final Map<String, String> templateVariableToValue = new HashMap<>();
+    log.info("URI Template: " + SINGLE_IMAGE_VERSION_URI_TEMPLATE.toString());
+    log.info("req uri: " + req.getRequestURI());
+    if (BASE_IMAGE_VERSION_URI.equals(req.getRequestURI())) {
+      log.info("Inside update image version");
```

Comment:
Nit: Too many unnecessary log lines. Can you please keep only essentials?

Reply:
Removed the info log as they were added for my debug purpose.